common/config/kafkaConfig.go

```diff
@@ -44,9 +44,8 @@ type (
 	// TopicConfig describes the mapping from topic to Kafka cluster
 	TopicConfig struct {
 		Cluster string `yaml:"cluster"`
-		// IsSecure describes whether the topic is secure, by default it is false
-		// If it is set to true, it allows to pass in the token to initialize secure producer for this topic
-		IsSecure bool `yaml:"isSecure,omitempty"`
+		// Properties map describes whether the topic properties, such as whether it is secure
+		Properties map[string]interface{} `yaml:"properties,omitempty"`
```

Comment:
we can use `any` instead of `interface{}`

Reply:
Updated, good to know it!